src/main/scala/com/workday/warp/common/WarpPropertyManager.scala

```diff
@@ -113,9 +113,10 @@ object WarpPropertyManager {
    */
   def logPropertyValues(): Unit = {
     val banner: StringBuilder = new StringBuilder
+    val usedConfigFile: String = if (new File(this.propertyFile).exists()) this.propertyFile else "none"
 
     // add WARP Framework Version and Configuration File in startup banner
-    banner ++= s"\nWARP Framework Version = '$version', Configuration File = '$propertyFile'"
+    banner ++= s"\nWARP Framework Version = '$version', Configuration File = '$usedConfigFile'"
```

Comment:
this will make the configuration banner log a bit less confusing, as we won't print the name of a non-existing file when there is no configuration file